core/frontend/src/DisplayStyleState.ts

```diff
@@ -796,18 +796,18 @@ export abstract class DisplayStyleState extends ElementState implements DisplayS
   public getIsBackgroundMapVisible(): boolean {
     return undefined !== this.iModel.ecefLocation && (this.viewFlags.backgroundMap || this.anyMapLayersVisible(false));
   }
-  public get backgroundMapElevationBias(): number {
+  public get backgroundMapElevationBias(): number | undefined{
```

Comment:
This is `@public`, can you document what it's for?

Reply:
Moved to internal.